content/en/history.md

```diff
@@ -5,9 +5,7 @@ category: 'About sigstore'
 position: 4
 ---
 
-Over 100 contributors have pushed over 2,800 commits to Sigstore since its founding. Contributors to the project represent over 20 different organizations, including Red Hat, Google, Chainguard, Purdue University, VMware, Twitter, Citi, Charm, Anchore, and Iron Bank. Today, Sigstore Cosign has over 2,300 GitHub Stars and Sigstore Rekor has more than 3 million log entries. Today, there are over 1,200 active members on the [public Slack channel](https://join.slack.com/t/sigstore/shared_invite/zt-mhs55zh0-XmY3bcfWn4XEyMqUUutbUQ), and many contributors and end users regularly attend Sigstore community meetings.
```

Comment:
I would prefer having some kind of intro paragraph to the history section, open to suggestions here.

Reply:
I don't think this is a good paragraph to keep as the opening section. Stats go out of date quickly, and I don't think most would care who is actively contributing.

The subsequent paragraph seemed sufficient on its own to provide historical context. Maybe another paragraph on why Sigstore was founded?

Reply:
If there is somewhere we should keep the stats up to date I think the docs could be the place to do that, as it should be a location of maintenance (aspirationally). I think having a "why" paragraph would be helpful. I would like this to be a longer section than it is but I'm not sure that we have a Sigstore historian keeping track yet. I can bring this up in a community meeting. Maybe we can update for now and think about how to position the history doc more?

Some recent data:
* 3,200 stars on Cosign repo
* 155 contributors on Cosign
* 6,600 commits across Cosign / Rekor / Fulcio / policy-controller / Gitsign (I know commits may not be the best thing to measure, but we can discuss growth over time)
* Over 9 million Rekor log entries (I am on bad wifi and will get a more precise number later)

FWIW, Google search pulls in the stats from this page so I think it would be nice to have them, and I can work to keep them updated more regularly.

Reply:
Sounds good to me!

Maybe we can link to dashboards on the LF insights page rather than inline stats? Then we don't need to worry about keeping stats up to date, and we get the benefit of nice dashboards too!